TryAtSoftware.Extensions.Collections.Tests/FluidDictionaryTests.cs

```diff
@@ -0,0 +1,78 @@
+namespace TryAtSoftware.Extensions.Collections.Tests;
+
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+public class FluidDictionaryTests
```

Comment:
In all `TryAtSoftware.*` projects we recommend using random values as much as possible. See `TryAtSoftware.Randomizer` - it exposes helper methods for generating random `string` or `int` values.

Reply:
This comment does not seem fixed to me.

Reply:
In order to get the random number, I have used `using TryAtSoftware.Extensions.Reflection.Tests.Randomization;`

Reply:
This is still open. All constant values that are currently being used should be randomized - this includes the keys and the values used in this and the next tests.